apps/test/unit/applab/designModeTest.js

```diff
@@ -165,4 +167,116 @@ describe('setProperty and read Property', () => {
       );
     });
   });
+
+  describe('changeThemeForCurrentScreen: ', () => {
```

Comment:
FWIW, coverage of the `changeThemeForCurrentScreen` method from this describe block is good.  Just a couple of the less common branching conditions at the end uncovered.

![image](https://user-images.githubusercontent.com/1615761/57484831-ecfce600-725e-11e9-916e-45011cb59b9d.png)